lib/ssh_scan/error.rb

```diff
@@ -0,0 +1,10 @@
+module SSHScan
+  class Error < Exception
```

Comment:
We might also consider adding a name space here like...

    module SSHScan
      module Exceptions
        class ConnectTimeout
          ....
        end
      end
    end